### PR TITLE
feat: Add support for Transifex translations in the LR MFE

### PR DESCRIPTION
**Pull request by justinhynes**
[APER-1419]

* Add Transifex configuration file
* Update existing Makefile to bring in line with other MFEs that support Transifex
* Update package.json and package-lock.json for i18n support

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com
+
+[edx-platform.your-resource-name-here]
```
Comment (MaxFrank13) [inline .tx/config]: Are we supposed to replace the `your-resource-name-here` with something that points to `frontend-app-learner-record`? I was just looking [at the one in `frontend-app-discussions`](https://github.com/openedx/frontend-app-discussions/blob/97b92a176241e4ce67ac0dafe31b31b61050f98e/.tx/config#L4)

Comment (justinhynes) [reply .tx/config]: 👀 good eyes! I noticed this bad copy-pasta as you were looking at it too.

Comment (MaxFrank13) [reply .tx/config]: The other MFEs have it formatted like this `o:open-edx:p:edx-platform:r:frontend-app-discussions`. Do we need that same prefix? I haven't been able to parse what it's doing

Comment (justinhynes) [reply .tx/config]: I think we'll need to update that when we move this repo to the open-edx org, but it's still in our org, so I don't think we need to do that.

Comment (justinhynes) [reply .tx/config]: Eh, I'll update it anywho. We can always fix it if needed.

**Review by MaxFrank13 — APPROVED** [review]

**Pull request merged**